extensions/shared/wrap-paid-block.jsx

```diff
@@ -14,7 +15,9 @@ export default ( { requiredPlan } ) =>
 		WrappedComponent => props => (
 			// Wraps the input component in a container, without mutating it. Good!
 			<Fragment>
-				<UpgradeNudge plan={ requiredPlan } blockName={ props.name } />
+				{ ! get( props, 'attributes.isBlockPreview', false ) && (
```

Comment:
I think optional chain and null coalesce work in Jetpack these days too (because we're on new enough `@automattic/calypso-build` tool (p4TIVU-9k9-p2).

So you can drop Lodash for simple operations like this and do this instead:

```js
props?.attributes?.isBlockPreview ?? false;
```

Reply:
Ah great! I didn't realise we could use that yet